README.md

```diff
@@ -1,5 +1,5 @@
-#  Microverse-Hello-Project:
-Microverse-Hello-Project is for testing knowledge of how to use git and github.
+#  Microverse-Project-hello:
+Microverse-Project-hello is for testing knowledge of how to use git and github.
 It contains a README file, html, css, gitignore and test.md
```

Comment:
- [ ] As the previous reviewer suggested, kindly use the Microverse template and just delete the placeholder and replace it with your project data. You can find the Microverse template [here](https://github.com/microverseinc/readme-template). 

- [ ] Well done for professional gitflow :clap:

Reply:
- [ ] [REMINDER] If you need to request for another review while you have reached the limit of reviewer request, you can request for an extra review :+1: